packages/sanity/playwright-ct/tests/formBuilder/PortableTextInput.spec.tsx

```diff
@@ -36,62 +36,55 @@ test.describe('Activation', () => {
 
 test.describe('Decorators', () => {
   test('Render default styles with keyboard shortcuts', async ({mount, page}, testInfo) => {
-    const {getModifierKey, focusPTE, typeInPTEWithDelay} = testHelpers({
-      page,
-      testInfo,
-    })
+    const {getModifierKey, getFocusedPortableTextEditor, insertPortableText, toggleHotkey} =
+      testHelpers({
+        page,
+        testInfo,
```

Comment:
**suggestion:** given that `testInfo` is no longer being used in `testHelpers`, consider removing it across the board (which should make these specs a tiny bit slimmer!)

Reply:
Yeah, I was thinking about that, but didn't want to change too much :)

I think the simpler we can keep this the better.

Reply:
Removed that dependency now.